src/Symfony/Installer/NewCommand.php

```diff
@@ -356,6 +358,22 @@ private function cleanUp()
     {
         $this->fs->remove(dirname($this->compressedFilePath));
 
+        try {
+            $licenseFile = array($this->projectDir.'/LICENSE');
+            $upgradeFiles = glob($this->projectDir.'/UPGRADE*.md');
+            $changelogFiles = glob($this->projectDir.'/CHANGELOG*.md');
```

Comment:
We want to maintain for example the `README.md` file (but changing its contents). Removing all the `.md` files seems too farsighted for me.